Task_1/Task_1/src/components/Input_form/form.jsx

```diff
@@ -0,0 +1,66 @@
+import { useState } from "react";
+
+const Form = (props) => {
+  const [enterTask, setEnterTask] = useState("");
+  const [enterDate, setEnterDate] = useState("");
+
+  function TaskHandler(event) {
+    setEnterTask(event.target.value);
+  }
+  function DateHandler(event) {
+    let date = event.target.value.toString();
```

Comment:
why we are storing date as string ? it can directly be Date.

Reply:
Implemented